microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshot-schema.md

```diff
@@ -303,8 +303,72 @@ The first number in the group of numbers for an edge in the `edges` array corres
 | Shortcut | An easier-to-read representation of some other path. This type is rarely used. For example, if you use `Function.prototype.bind` to create a bound function with some bound arguments, V8 creates a `JSBoundFunction`, which points to a `FixedArray` (an internal type), which points to each bound argument. When producing a snapshot, V8 adds a shortcut edge from the bound function directly to each bound argument, bypassing the `FixedArray`. |
 | Element | Object properties where the key is a number. |
 
+#### locations
+
+The `locations` array, which is at the top level of the `.heapsnapshot` data, contains information about the location of `HeapSnapshot` nodes.
+
+To use this array, the following information is needed:
```

Comment:
Before going into this, I think we miss a sentence that explains that the locations array is (like the nodes array) a continuous list of numbers which is meant to be read in groups of 4 (or, really, in groups of `location_fields.length` to be safe). 
The example you added later shows this very well, but I think we need to quickly introduce this concept here before.

Maybe something like:

The `locations` array consists of a series of numbers meant to be read by groups of size `snapshot.meta.location_fields.length`.

Reply:
Good catch, I added it.